python/src/ufo.c

```diff
@@ -139,6 +139,36 @@ static PyMethodDef exported_methods[] = {
     {NULL, NULL, 0, NULL}
 };
 
+#if PY_MAJOR_VERSION >= 3
```

Comment:
Why not drop support for Python 2 entirely?

Reply:
Because of our old servers and the lack of my motivation to keep two branches up to date. It will happen though when concert3 is there...

Reply:
Makes (some) sense :D